homeassistant/components/google_assistant_sdk/helpers.py

```diff
@@ -27,6 +27,7 @@ async def async_send_text_commands(commands: list[str], hass: HomeAssistant) ->
         raise err
 
     credentials = Credentials(session.token[CONF_ACCESS_TOKEN])
-    with TextAssistant(credentials) as assistant:
+    language_code = entry.options.get(CONF_LANGUAGE_CODE, DEFAULT_LANGUAGE_CODE)
```

Comment:
Can the default language use the home assistant language?

I was just looking at the conversation API and I saw that it takes a language as input, but also uses the home assistant language as a default and that seemed interesting to take advantage of to simplify configuration for users.  At the same time, this is a service call so it seems like it may not always be clear to the user how language works...

Reply:
Done. Used language and country to get the default language code.